AssetInformationApi/V1/Controllers/AssetInformationApiController.cs

```diff
@@ -134,6 +134,32 @@ public async Task<IActionResult> PatchAsset([FromRoute] EditAssetByIdRequest que
             }
         }
 
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPatch]
+        [Route("{id}/address")]
+        [LogCall(LogLevel.Information)]
+        public async Task<IActionResult> PatchAssetAddress([FromRoute] EditAssetByIdRequest query, [FromBody] EditAssetRequest asset)
+        {
+            var bodyText = await HttpContext.Request.GetRawBodyStringAsync().ConfigureAwait(false);
+            var ifMatch = GetIfMatchFromHeader();
+            var token = _tokenFactory.Create(_contextWrapper.GetContextRequestHeaders(HttpContext));
+            try
+            {
+                var result = await _editAssetUseCase.ExecuteAsync(query.Id, asset, bodyText, token, ifMatch).ConfigureAwait(false);
+
+                if (result == null) return NotFound();
+
+                return Ok();
```

Comment:
The response should be 201 seeing as we are not returning anything

Reply:
Changed